datasets/ua/nsdc_sanctions/ua_nsdc_sanctions.yml

```
@@ -31,56 +31,56 @@ data:
+        value: "\"Платежный центр\""
+      - match: "Tovarystvo z obmezhenoiu vidpovidalnistiu Rozrakhunkova nebankivska kredytna orhanizatsiia \"Platezhnyi tsentr\" (Obshchestvo s ohranychennoi otvetstvennostiu Raschёtnaia nebankovskaia kredytnaia orhanyzatsyia \"Platezhnyi tsentr\", Tovarystvo z obmezhenoiu vidpovidalnistiu Rozrakhunkova nebankivska kredytna orhanizatsiia \"Platizhnyi tsentr\")"
+        value: "\"Platezhnyi tsentr\""
+      - match: "Платіжна організація міжнародної платіжної системи \"Колібрі\", якою управляє публічне акціонерне товариство \"Сбербанк Росії\" (Платіжна організація міжнародної платіжної системи \"Колібрі\", якою керує публічне акціонерне товариство \"Ощадбанк Росії\", публичное акционерное общество \"Сбербанк России\", стара назва – \"Бліц\")"
+        value: "\"Платіжна організація міжнародної платіжної системи \"Колібрі\""
+      - match: "Platizhna orhanizatsiia mizhnarodnoyi platizhnoyi systemy \"Kolibri\", yakoiu upravliaie publichne aktsionerne tovarystvo \"Sberbank Rosiyi\" (Platizhna orhanizatsiia mizhnarodnoyi platizhnoyi systemy \"Kolibri\", yakoiu keruie publichne aktsionerne tovarystvo \"Oshchadbank Rosiyi\", publychnoe aktsyonernoe obshchestvo \"Sberbank Rossyy\", stara nazva – \"Blits\")"
+        value: "\"Platizhna orhanizatsiia mizhnarodnoyi platizhnoyi systemy \"Kolibri\""
+      - match: "Товариство з обмеженою відповідальністю \"Транспортні автоматизовані інформаційні системи – ТАЇС\" (Общество с ограниченной ответственностью \"Транспортные автоматизированные информационные системы – ТАИС\", Товариство з обмеженою відповідальністю \"Транспортні автоматизовані інформаційні системи – ТАЇС\")"
+        value: "\"Транспортні автоматизовані інформаційні системи – ТАЇС\""
+      - match: "Tovarystvo z obmezhenoiu vidpovidalnistiu \"Transportni avtomatyzovani informatsiini systemy – TAIS\" (Obshchestvo s ohranychennoi otvetstvennostiu \"Transportnye avtomatyzyrovannye ynformatsyonnye systemy – TAYS\", Tovarystvo z obmezhenoiu vidpovidalnistiu \"Transportni avtomatyzovani informatsiini systemy – TAYiS\")"
+        value: "\"Transportni avtomatyzovani informatsiini systemy – TAIS\""
+      - match: "Телеканали \"РТР-Планета\", інформаційний канал \"Росія-24\", які входять до структури Федерального державного унітарного підприємства \"Всеросійська державна телевізійна і радіомовна компанія\" (Телеканали \"РТР-Планета\", інформаційний канал \"Росія-24\", які входять до структури Федерального державного унітарного підприємства \"Всеросійська державна телевізійна і радіомовна компанія\", Телеканалы \"РТР-Планета\", информационный канал \"Россия-24\", которые входят в структуру Федерального государственного унитарного предприятия \"Всероссийская государственная телевизионная и радиовещательная компания\")"
+        value: 
```

I'm pretty sure where the replacement is an array, the key has to be `values`.

```suggestion
        values: 
```